README.md

```diff
@@ -107,9 +107,23 @@ item 0: User {
 [![Javadocs](http://www.javadoc.io/badge/com.spotify/hamcrest-jackson.svg?color=blue)](http://www.javadoc.io/doc/com.spotify/hamcrest-jackson)
 
 Similar to the POJO matchers, the JSON matchers let you describe a
-JSON structure and match against it.
+JSON structure and match a string or against it.
```

Comment:
I agree this quite confusing. I will reword. 

The current docs talk about "JSON structure" but it means "JsonNode". I'll try to clarify that.